prime-router/src/main/kotlin/metadata/LivdLookup.kt

```diff
@@ -0,0 +1,217 @@
+package gov.cdc.prime.router.metadata
+
+import gov.cdc.prime.router.Metadata
+
+object LivdLookup {
+    val livdTableName = "LIVD-SARS-CoV-2"
+    /**
+     * Will find a value in the LIVD table based on the rest kit name id, equipment model name, equipment model id, or
+     * the device id
+     * @param testPerformedCode used for additional filtering of the test info in case we are dealing with tests that
+     * check for more than on type of disease
+     * @param processingModeCode will flag the data as test data which will make it ignore any test devices
+     * @param deviceId the ID of the test device to lookup LIVD information by
+     * @param equipmentModelId the ID of the equipment model to lookup LIVD information by
+     * @param testKitNameId the ID of the test kit to lookup LIVD information by
+     * @param equipmentModelName the name of the equipment model to lookup LIVD information by
+     * @param tableColumn the name of the table column to lookup LIVD information on
+     * @param tableRef allows us to pass in the reference to the table, just used for testing purposes
+     * @return a possible String? value based on the lookup
+     */
+    fun find(
+        testPerformedCode: String?,
+        processingModeCode: String?,
+        deviceId: String?,
+        equipmentModelId: String?,
+        testKitNameId: String?,
+        equipmentModelName: String?,
+        tableColumn: String,
+        tableRef: LookupTable? = Metadata.getInstance().findLookupTable(name = livdTableName)
+    ): String? {
+        if (tableRef == null) {
+            error("Schema Error: could not find table '$tableRef'")
```

Comment:
At this point, you do not know if it is a schema error as this function could be called from anywhere. Remove the Schema error part.